packages/docs-site/.firebaserc

```diff
@@ -1,21 +1,22 @@
 {
   "projects": {
-    "docs-site": "standards"
+    "production": "standards",
+    "staging": "standards-staging"
   },
   "targets": {
-    "docs-site": {
+    "standards": {
```

Comment:
Not sure `standards` is actually the right name here. I know it's picky but we use `standards` as the name of the project in its entirety. `docs-site` probably is the right name, plus it matches the package name in the repo.

Reply:
This is because i'm targeting the 'standards' project within Firebase. Under each of these projects (standards and standards-staging) there are multiple sites deployed:

docs-site
react-storybook
vue-storybook
...

Reply:
ah, ignore me then :)